Mathlib/Algebra/Polynomial/Inductions.lean

```diff
@@ -153,6 +153,7 @@ if it holds for
 * `a * X`, with `a ∈ R`,
 * `p * X`, with `p ∈ R[X]`,
 * `p + a`, with `a ∈ R`, `p ∈ R[X]`,
+
 with appropriate restrictions on each term.
```

Comment:
How do situations like this get rendered? The `with appropriate...` shouldn't be a new paragraph here; it should be a continuation of the previous one

Reply:
I think it gets rendered fine. (Most of this PR's changes don't affect the actual rendering - but they make it obvious that things are fine, and make the cases where it *does* matter easier to catch.)

Reply:
I've commented [on zulip](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Style.20.3Abicycle.3A.20.3A.20indenting.20second.20lines.20in.20doc-strings/near/513150580): always separating the further code from the bullet points makes it *much* easier to write a linter for the things this PR detects.

Reply:
I have removed the blank line changes for now.